### ISSUE TITLE
[Modular CLI]: Installing mojo does not work

### ISSUE THREAD

**Issue by DragonSeason** [modularml/mojo#931]
### Issue description

I followed the instructions for windows on this site:
https://developer.modular.com/download
And it seems like I can't proceed past the 4th step.
My internet connection is not the cause of the timeout.
The following is the output I get:

```
# Found release for https://packages.modular.com/mojo @ 0.3.0, installing to /home/maynore/.modular/pkg/packages.modular.com_mojo
modular: error: failure detected - please run `modular clean` and try again
==========================
Failure Information:
        - http error: failed to reach URL https://cas.modular.com/0/HT-HgQXEfA3jVHcFNzgYkKzozY_G9-LICgmyGqODtus with cURL error Timeout was reached - Looking for HT-HgQXEfA3jVHcFNzgYkKzozY_G9-LICgmyGqODtus
        - http error: failed to reach URL https://cas.modular.com/0/0V2Bqdr8786hJwh6FzhEwXzE0ApqWoFaNETsvu7zm4E with cURL error Timeout was reached - Looking for 0V2Bqdr8786hJwh6FzhEwXzE0ApqWoFaNETsvu7zm4E
        - http error: failed to reach URL https://cas.modular.com/0/r7Uh_7YudP025MOQM_uMTbA5iOdIlKqnoXgfywB20_Y with cURL error Timeout was reached - Looking for r7Uh_7YudP025MOQM_uMTbA5iOdIlKqnoXgfywB20_Y
        - http error: failed to reach URL https://cas.modular.com/0/hX6EBf6V1on6XH2SsLN4lJSPvWalVxLDd7_H9pHPChA with cURL error Timeout was reached - Looking for hX6EBf6V1on6XH2SsLN4lJSPvWalVxLDd7_H9pHPChA
==========================
```

### Steps to reproduce

See the steps to install for windows in this instruction:
https://developer.modular.com/download

### Version Info

```shell
Modular CLI version: modular 0.1.4 (6b54d308)
```

**Comment by SimplyYummy** [comment]
Duplicate of: https://github.com/modularml/mojo/issues/598
See: https://docs.modular.com/mojo/roadmap.html#mojo-sdk-known-issues
See: https://www.modular.com/newsletters/modverse-weekly-2

```
[#598: Failed to reach URL timeout was reached](https://github.com/modularml/mojo/issues/598) requires connection speed of 600kbs+ to download before timeout. Working on compressing artifacts and adding a download bar.
```

In the meantime try running `modular clean` and then `modular install mojo`

**Comment by SimplyYummy** [comment]
@DragonSeason this hopefully should be fixed with the new 0.3.1 release. Try the steps here https://docs.modular.com/mojo/changelog.html#v0.3.1-2023-09-28
```
sudo apt-get update
sudo apt-get install modular
modular clean
modular install mojo
```
